@AH-64D Apache Longbow/Addons/fza_ah64_dms/functions/eden/fn_edenPointNext.sqf

```diff
@@ -1,22 +1,26 @@
+#include "\fza_ah64_dms\headers\constants.h"
+params ["_entity", "_type"];
 
-params ["_type"];
+private _usedIndex = [];
+private _index = -1;
+private _fzalogic = allMissionObjects "Logic" select { 
```

Comment:
Can't you just select `allMissionObjects "fza_dms_point_xyz"` to cut out the need for this filtering?

Reply:
Done